docs/poc/poc.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: default
 title: POC migrations
-nav_order: 8
+nav_order: 9
```

Comment:
nit: These nav orders are prone to errors. We should look at a way to automate generation of these